src/content/cloud/ssl.mdx

```diff
@@ -71,4 +71,16 @@ spec:
         path: /
 ```
 
-We recommend you follow this option. This way your ingress configuration can be deployed on any Okteto Cloud namespace.
\ No newline at end of file
+We recommend you follow this option. This way your ingress configuration can be deployed on any Okteto Cloud namespace.
+
+## Customizing the endpoints shown in the Okteto UI
+
+Okteto provides additional annotations that enable you to customize the endpoints shown in the Okteto UI.
+
+To hide specific endpoints from the Okteto UI, you can use the `dev.okteto.com/hide-from-ui: true` annotation.
+Simply apply this annotation to your `Ingress` or `VirtualService` resource, and the endpoints associated with that resource will not be displayed in the Okteto UI.
+
+If you want to replace the default endpoints for an `Ingress` or `VirtualService` resource, you can use the `dev.okteto.com/endpoints` annotation.
+This annotation allows you to define a list of endpoints, separated by commas, that will replace the default endpoints.
+Additionally, the value of this annotation expands the `OKTETO_NAMESPACE` and `OKTETO_DOMAIN` environment variables, providing an easy way to make your endpoints portable between namespaces.
```

Comment:
Can we add example code for these?

Also, will this work for docker compose files? If yes, please include a sample too

Reply:
added sample